Add keyboard shortcuts to the Config/Keybinding chapter. (#875)
ellie/atuin: docs/docs/config/key-binding.md

bind \cr _atuin_search
bind -M insert \cr _atuin_search
```

## Atuin UI shortcuts

| Shortcut                                  | Action                                                                                                             |
| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
| ctrl + r                                  | Cycle through filter modes                                                                                         |

> | ctrl + s                                  | Switch the search mode, changes the engine and possibly other interface settings                                   |

Author: done! thanks:)

| ctrl + a / home                           | Move the cursor to the start of the line                                                                           |
| ctrl + e / end                            | Move the cursor to the end of the line                                                                             |
| ctrl + backspace / ctrl + alt + backspace | Remove the previous word / remove the word just before the cursor                                                  |
| ctrl + delete / ctrl + alt + delete       | Remove the next word or the word just after the cursor                                                             |
| ctrl + w                                  | Remove the word before the cursor even if it spans across the word boundaries                                      |
| ctrl + u                                  | Clear the current line                                                                                             |
| ctrl + n / ctrl + j                       | Select the next item on the list                                                                                   |

> | ctrl + p / ctrl + k / up arrow            | Select the previous item on the list                                                                               |

Reviewer: `up arrow` is inconsistently named in relation to the other arrow key lines

Author: Thank you! 
I tried to replace UTF symbols with emojis. But on my mac and in the web Right arrow looks diffrent from others. :(

| page down                                 | Scroll search results one page down                                                                                |
| page up                                   | Scroll search results one page up                                                                                  |
| enter                                     | Select highlighted item                                                                                            |

> | ↓                                         | Return original or return query depending on settings if no entries are found, or select the next item on the list |

Reviewer: Maybe we can duplicate the line here to be a bit simpler
```suggestion
| ↓ (with no entry selected)                | Return original or return query depending on settings                                                              |
| ↓                                         | Select the next item on the list                                                              |
```
(this last line can be merged with the `ctrl + n / ctrl + j` line)
